src/Containers/PlansListView.js

```diff
@@ -90,19 +90,27 @@ export default class PlansListView extends Component {
     // https://expo.github.io/vector-icons/
 
     static navigationOptions = ({navigation}) => ({
-            // header: null,
-            title: 'Budget Plans',
-            headerRight: (
-                <Icon 
-                    name='plus-square'
-                    size={30}
-                    onPress={ () => navigation.navigate('PlanCreate') }                
-                    backgroundColor= "rgba(0,0,0,0)"
-                    // color="rgba(0,122,255,1)"
-                    containerStyle={{margin: 50}}
-                />
-            )});
-        
+    // header: null,
+        title: 'Budget Plans',
+        headerRight: (
+            <Icon 
+                name='plus-square'
+                size={30}
+                onPress={ () => navigation.navigate('PlanCreate', { handleUpdate: this.handleUpdate.bind(this) }) }                
+                backgroundColor= "rgba(0,0,0,0)"
+                // color="rgba(0,122,255,1)"
+                containerStyle={{margin: 50}}
+            />
+    )});
+
```

Comment:
Try this:
1. add this line to `componentDidMount`:
```
componentDidMount() {
  this.props.navigation.setParams({ handleUpdate: this.handleUpdate });
}
```
2. change the `navigationOptions` to something like:
```
static navigationOptions = ({navigation, state}) => ({
    // header: null,
        title: 'Budget Plans',
        headerRight: (
            <Icon 
                name='plus-square'
                size={30}
                onPress={ () => navigation.navigate('PlanCreate', { handleUpdate: state.params.handleUpdate }) }                
                backgroundColor= "rgba(0,0,0,0)"
                // color="rgba(0,122,255,1)"
                containerStyle={{margin: 50}}
            />
    )});
```